src/components/appSwitch/AppSwitch.js

```diff
@@ -110,6 +111,8 @@ const AppSwitch = (props: LoadingProps) => {
    * @returns {Promise<void>}
    */
   const initialize = async () => {
+    //initialize logged in user feed and stuff
+    await userStorage.initRegistered()
```

Comment:
from my point of view, analytics shouldn't know about user storage and other stuff
it's a separate module. let it exports identifyWith() and we'll call it with some creds (id, email)

i propose to move 115/187 lines to the init.js

Reply:
this is exactly what i did here. i remove identifywith..(wallet,userstorage)
and I call identifyWith(id,email) in line 187.
identifyWith for signup router happens in signupstate
and identifyWith for signedin user happens here
init.js happens on both

Reply:
> this is exactly what i did here. i remove identifywith..(wallet,userstorage)
> and I call identifyWith(id,email) in line 187.
> identifyWith for signup router happens in signupstate
> and identifyWith for signedin user happens here
> init.js happens on both

my bad, didn't pay attention that it's AppSwitch, thought it's analytics.js